content/pages/health-care/health-conditions/mental-health.md

```diff
@@ -17,7 +17,7 @@ If you qualify for VA health care, you can get high-quality mental health servic
 
 <div class="usa-alert usa-alert-warning va-alert">
   <div class="usa-alert-body">
-	<h5>Are you in crisis? <a id="crisis-expander-link">You have options.</a></h5>
+	<h4 class="usa-alert-heading">Are you in crisis? <a id="crisis-expander-link">You have options.</a></h4>
```

Comment:
Can we get a little more padding beneath, and/or shrink the icon a bit? 
<img width="741" alt="screen shot 2017-11-17 at 10 58 42 am" src="https://user-images.githubusercontent.com/4895604/32956271-51c182fc-cb86-11e7-8603-7c05ddc14ba4.png">